src/main/java/org/prebid/server/cache/model/DebugHttpCall.java

```diff
@@ -6,19 +6,15 @@
 /**
  * Holds HTTP interaction related data.
  */
-@AllArgsConstructor(staticName = "of")
 @Value
-public class CacheHttpCall {
-
-    private static final CacheHttpCall EMPTY = CacheHttpCall.of(null, null, null);
+@AllArgsConstructor(staticName = "of")
+public class DebugHttpCall {
 
     CacheHttpRequest request;
```

Comment:
Is it make sense to replace the type to `org.prebid.server.bidder.model.HttpRequest` ?
Nothing cache specific here in this model.